fourier/internal/fftpack/fftpack_test.go

```diff
@@ -616,3 +616,827 @@ var cfftTests = []struct {
 		wantiifac: []int{2, 1, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
 	},
 }
+
+func TestSint(t *testing.T) {
```

Comment:
Could you please add comments to the sections of the tests to make it more clear what's going on? (This goes for the other added tests here)

Reply:
Is it OK if I just say that these tests reiterate the tests that are part of FFTPACK?

Reply:
I'll add those, but it does seem odd to me to do it here when the tests that exist in testblas and to a lesser extent testlapack largely don't have any commentary.

Reply:
Do you have particular functions in mind? I did a quick survey and most of the float64 blas tests are only one section, and of the blas/lapack functions that have multiple sections typically those sections have some amount of explanation text.

Reply:
This is always going to be a subjective assessment, but here is a list of tests in blas and lapack that look like they are commented to a similar level. There are others where there is commentary, but at least when I read it, I need to do a lot of work to understand what is being done - this is obviously an interaction between my level of linalg experience and the commentary. I've left out tests where there are tables or the tests are trivial.

blas: Zgbmv, Zhbmv, Ztbmv, Ztpsv, Ztrsv
lapack: Debgd2, Dgelqf, Dgeqp3, Dlags2, Dlange, Dlaln2, Dlanst, Dlansy, Dlanv2, Dlapll, Dlaqr23, Dlarfx, Dlartg, Dlasr, Dlatrd, Dorg2r, Dorgbr, Dorgtr, Dpotf2, Dtrtri2, Dtrtri.

It's worth noting that many of the tests explain what they do in the `Errorf` format string, though not necessarily how they get there - I think the fftpack tests do both (intentionally).

Reply:
Thanks for the list. I'll go through and try to improve.